jcl/src/jdk.attach/share/classes/com/ibm/tools/attach/attacher/OpenJ9VirtualMachine.java

```diff
@@ -277,6 +282,19 @@ public Properties getThreadInfo() throws IOException {
 		return IPC.receiveProperties(responseStream, true);
 	}
 
+	/**
+	 * Execute a diagnostic command on a target VM.
+	 * 
+	 * @param diagnosticCommand name of command to execute
+	 * @return properties object containing serialized result
+	 * @throws IOException in case of a communication error
+	 */
+	public Properties executeDiagnosticCommand(String diagnosticCommand) throws IOException {
+		IPC.logMessage("enter executeDiagnosticCommand ", diagnosticCommand); //$NON-NLS-1$
```

Comment:
unwanted space at end of string

Reply:
See above.

Reply:
I think I saw places where that space is missing.